### ISSUE TITLE
Auto change the number of games per line

### ISSUE THREAD

**Issue by zwrawr** [ludumdare/ludumdare#1363]
*(Im on a 1440p screen so, these screen shots don't show off the issues very well...)*

### Older phone screens (600px)
![image](https://user-images.githubusercontent.com/18352787/32786482-f847ec48-c94b-11e7-96ee-2820bc6707b3.png)
This should probably be 1 per row

### When the screen is narrow (900px)
![image](https://user-images.githubusercontent.com/18352787/32786351-97376730-c94b-11e7-9804-64bece65f5eb.png)
This should probably be 2 per row, maybe even 3 per row.

### When the screen is normal (1920px)
![image](https://user-images.githubusercontent.com/18352787/32786228-44308d0a-c94b-11e7-8dcc-4d12976f71aa.png)
This should probably be three or four.

### When the screen is wide (2,560px)
![image](https://user-images.githubusercontent.com/18352787/32786264-5a475326-c94b-11e7-8ced-b391ccec400b.png)
I'd go for 5 or 6 for wider than normal screens

**Comment by povrazor** [comment]
Yeah, I asked @local-minimum to get width controls working (see the tiny gear hiding between the filters bar and games themselves). Mobile vs Desktop needs different defaults, and IMO there's no reason to be able to select 1-wide on Desktop.

**Comment by zwrawr** [comment]
This is not implemented yet, issue should be re opened.

**Comment by mikekasprzak** [comment]
Oh sorry, I missed the word "auto".
